src/app/package-json.js

```diff
@@ -268,6 +272,8 @@ const packageJSON = (current, context) => {
 
   // Support the documentation tooling option.
   if (context.docs) {
+    result.files.push('docs');
```

Comment:
Why? It seems to me the package should only require those things needed to use it.